cloudvolume/cloudvolume.py

```diff
@@ -1032,7 +1033,7 @@ def __getitem__(self, slices):
 
     return self._boss_cutout(requested_bbox, steps, channel_slice)
 
-  def download_point(self, pt, size=256, mip=None):
+  def download_point(self, pt, size=256, mip=None, allow_cropped=False):
```

Comment:
CloudVolume already has a mode called "autocrop" that does this. Does it make sense to have a separate parameter for this? Not a rhetorical question.

Reply:
Autocrop is cool! Didn't know that was a thing. Don't think we need this flag then. I think I found an autocrop bug though, I'll open an issue